lib/chore/strategies/consumer/single_consumer_strategy.rb

```diff
@@ -10,16 +10,18 @@ def initialize(fetcher, opts={})
       end
 
       # Begins fetching from the configured queue by way of the configured Consumer. This can only be used if you have a
-      # single queue which can be kept up with at a relatively low volume. If you have more than a single queue configured,
-      # it will raise an exception.
+      # single queue which can be kept up with at a relatively low volume. If you have more than a single queue
+      # configured, it will raise an exception.
+      #
+      # @return [TrueClass, FalseClass]
```

Comment:
No return doc needed here (I'll stop repeating myself -- just need to apply this across the doc changes you've made)